package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "v1sdk",
-  "version": "1.0.9",
+  "version": "1.0.10",
```

Comment:
This is not a patch, this is a new minor version (new feature without breaking changes).